docs/serving/using-a-tls-cert.md

```diff
@@ -40,10 +40,8 @@ use any certificate from a CA that supports the ACME protocol. However, you must
 use and configure your certificate issuer to use the
 [`DNS-01` challenge type](https://letsencrypt.org/docs/challenge-types/#dns-01-challenge).
 
-> **Important:** Certificates issued by Let's Encrypt are valid for only [90
-> days][le-faqs]. Therefore, if you choose to manually obtain and configure your
-> certificates, you must ensure that you renew each certificate before it
-> expires.
+!!! warning
+    Certificates issued by Let's Encrypt are valid for only [90days][le-faqs]. Therefore, if you choose to manually obtain and configure your certificates, you must ensure that you renew each certificate before it expires.
```

Comment:
Feels like we should have some formatting for things like Let's Encrypt that are a _proper_name?

Reply:
Proper nouns feel like simple grammar, maybe not needed for Style Guide?